apollo-backend/src/main/java/io/logz/apollo/controllers/BlockerDefinitionController.java

```diff
@@ -188,9 +192,18 @@ public void updateBlockerDefinition(int id, String name, Integer environmentId,
         blockerDefinition.setStackId(stackId);
         blockerDefinition.setAvailability(availability);
         blockerDefinition.setBlockerTypeName(blockerTypeName);
-        blockerDefinition.setBlockerJsonConfiguration(blockerJsonConfiguration);
         blockerDefinition.setActive(isActive);
 
+        if (blockerTypeName.equals(BlockerTypeName.SINGLE_REGION)) {
+            blockerJsonConfiguration = initSingleRegionBlockerJsonConfig(environmentId, serviceId, stackId, availability, blockerJsonConfiguration, req);
+            if (blockerJsonConfiguration == null) return;
+            blockerDefinition.setBlockerJsonConfiguration(blockerJsonConfiguration);
+        } else {
+            if (blockerJsonConfiguration != null && !blockerJsonConfiguration.equals("{}")) {
+                blockerDefinition.setBlockerJsonConfiguration(blockerJsonConfiguration);
+            }
+        }
```

Comment:
I wouldn't change the existing behavior before discussing it with the team

Reply:
it is just did not worked at all - there is no change of behavior .

Reply:
@maorshap need to be sure that you can still update blocker definitions from UI (not only SRB).
Please check it locally before merging to master

Reply:
@matvey-mtn I checked